packages/app/src/LogSidePanel.tsx

```diff
@@ -1648,6 +1660,24 @@ function PropertySubpanel({
                     </Button>
                   </Link>
                 ) : null}
+
+                {!!toggleColumn && keyPath.length === 1 ? (
+                  <Button
+                    className="fs-8 text-muted-hover child-hover-trigger p-0"
+                    variant="link"
+                    as="a"
+                    title={`${
+                      displayedColumns?.includes(keyPathString)
+                        ? 'Remove'
+                        : 'Add'
+                    } ${keyPathString} column`}
```

Comment:
```suggestion
                    } ${keyPathString} column from results table`}
```
Nit: Would make it a tiny bit more clear what the button does, though the table icon should help!